src/pages/SigningOfficialConsole.js

```diff
@@ -0,0 +1,383 @@
+  };
+
+  componentDidMount() {
+    let currentUser = Storage.getCurrentUser();
+    this.setState({
+      currentUser: currentUser
+    }, () => {
+      this.init(currentUser);
+    });
+  }
+
+  init(currentUser) {
+
+    PendingCases.findConsentPendingCasesByUser(currentUser.dacUserId).then(
+      duls => {
+        this.setState(prev => {
+          prev.electionsList.dul = duls.dul;
+          prev.totalDulPendingVotes = duls.totalDulPendingVotes;
+          return prev;
+        });
+      }
+    );
+
+    PendingCases.findDataRequestPendingCasesByUser(currentUser.dacUserId).then(
+      dars => {
+        this.setState(prev => {
+          prev.electionsList.access = dars.access;
+          prev.totalAccessPendingVotes = dars.totalAccessPendingVotes;
+          return prev;
+        });
+      }
+    );
+  }
+
+  openDULReview = (voteId, referenceId) => (e) => {
+    this.props.history.push(`dul_review/${voteId}/${referenceId}`);
+  };
+
+  isDuLCollectEnabled = (pendingCase) => {
+    const pendingCaseDulCollectStatus = (pendingCase.alreadyVoted === true);
+    const dacId = _.get(pendingCase, 'dac.dacId', 0);
+    // if the pending case doesn't have a DAC, then any chair should be able to collect votes:
+    if (dacId === 0) { return pendingCaseDulCollectStatus; }
+    const dacChairRoles = _.filter(this.state.currentUser.roles, { 'name': USER_ROLES.chairperson, 'dacId': dacId });
+    return (!_.isEmpty(dacChairRoles)) && pendingCaseDulCollectStatus;
+  };
+
+  openDulCollect = (consentId) => (e) => {
+    this.props.history.push(`dul_collect/${consentId}`);
+  };
+
+  openFinalAccessReview = (referenceId, electionId, rpElectionId) => (e) => {
+    this.props.history.push(`${'final_access_review'}/${referenceId}/${electionId}`);
+  };
+
+  openAccessReview = (referenceId, voteId, rpVoteId) => async (e) => {
+    const pathStart = await NavigationUtils.accessReviewPath();
+    if (rpVoteId !== null) {
+      this.props.history.push(`${pathStart}/${referenceId}/${voteId}/${rpVoteId}`);
+    } else {
+      this.props.history.push(`${pathStart}/${referenceId}/${voteId}`);
+    }
+  };
+
+  isAccessCollectEnabled = (pendingCase) => {
+    const pendingCaseAccessCollectStatus =
+      (pendingCase.alreadyVoted === true) &&
+      (!pendingCase.isFinalVote) &&
+      (pendingCase.electionStatus !== 'Final');
+    const dacId = _.get(pendingCase, 'dac.dacId', 0);
+    // if the pending case doesn't have a DAC, then any chair should be able to collect votes:
+    if (dacId === 0) { return pendingCaseAccessCollectStatus; }
+    const dacChairRoles = _.filter(this.state.currentUser.roles, { 'name': USER_ROLES.chairperson, 'dacId': dacId });
+    return (!_.isEmpty(dacChairRoles)) && pendingCaseAccessCollectStatus;
+  };
+
+  openAccessCollect = (referenceId, electionId) => (e) => {
+    this.props.history.push(`access_collect/${electionId}/${referenceId}`);
+  };
+
+
+  handleOpenModal() {
+    this.setState({ showModal: true });
+  }
```

Comment:
Back on line 30, you have this: `this.handleOpenModal = this.handleOpenModal.bind(this);`
That's only necessary because of the construction of this method. You could eliminate lines 30-31 if you changed these two modal methods like this:

```
  handleOpenModal = () => {
    this.setState({ showModal: true });
  }
```
and 
```
  handleCloseModal = () => {
    this.setState({ showModal: false });
  }
```